baseapp/baseapp.go

```diff
@@ -115,7 +115,7 @@ func (app *BaseApp) Name() string {
 // SetCommitMultiStoreTracer sets the store tracer on the BaseApp's underlying
 // CommitMultiStore.
 func (app *BaseApp) SetCommitMultiStoreTracer(w io.Writer) {
-	app.cms.WithTracer(w)
+	app.cms.GetTracer().Writer = w
```

Comment:
The initial API call was bad and that's my fault. Can we just do `app.cms.SetTracer(w)`?